internal/resource/config/captchaproviders/settings/captcha_provider_settings_resource.go

```diff
@@ -0,0 +1,13 @@
+package captchaproviderssettings
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+func (m *captchaProviderSettingsResourceModel) setNullObjectValues() {
```

Comment:
This shouldn't be necessary after recent updates to the code generator - it should be using the emptyModel method that gets generated in the resource_gen file.

Reply:
fixed